README.md

```diff
@@ -1,59 +1,49 @@
+![image](https://user-images.githubusercontent.com/24784631/216797451-f3d58b02-c900-4d37-9a3c-03ce48ffc411.png)
+
+## Table of Contents
 * [Introduction](#introduction)
 * [Requirements](#requirements)
 * [Install](#install)
 * [Data Files](#data-files)
+* [Common Issues](#common-issues)
 * [Reporting issues](#reporting-issues)
 * [Submitting fixes](#submitting-fixes)
-* [Copyright](#copyright)
-
-## Introduction
-
-LegionCore is an *MMORPG* Framework for Legion (26972) based on the uwow.biz core leaked early 2020.
-LegionCore itself has been derived from an old version of TrinityCore.
-
-TrinityCore is a *MMORPG* Framework based mostly in C++.
-
-TrinityCore is derived from *MaNGOS*, the *Massive Network Game Object Server*, and is
-based on the code of that project with extensive changes over time to optimize,
-improve and cleanup the codebase at the same time as improving the in-game
-mechanics and functionality.
-
-LegionCore development is completely open source; community involvement is highly encouraged.
-
-
-## Requirements
+* [Thank you](#thank-you)
 
-The software requirements for LegionCore are very similar to the one from TrinityCore.
-The main difference worth mentioning is that on Windows currently BOOST version 1.64 is required.
+# Introduction
+LegionCore is a **RPGMMO** framework for WOW Legion *(Build 26972)*. This core is based off of the UWOW core leak of 2020. Which was derived from an old version of [TrinityCore](https://github.com/TrinityCore/TrinityCore). LegionCore is completely opensource and is developed by the community. To submit a pull request please follow this template [here](submitting-fixes).
 
-Software requirements are available in the [wiki](https://www.trinitycore.info/display/tc/Requirements) for
-Windows, Linux and macOS.
+If you want you can join the community discord: [here](https://discord.gg/uaP2aeJ7sj).
 
+# Requirements
+ 
+ ### Important
+ The main difference to note is that LegionCore requires Boost (1.64.0), Visual Studio 2017 & 2017 build tools, and MySQL (5.6.51).
+ 
+[Windows specific](https://www.trinitycore.info/en/install/requirements/windows)
+  
+[Linux specific](https://www.trinitycore.info/en/install/requirements/linux)
 
-## Install
+[Mac specific](https://www.trinitycore.info/en/install/requirements/macos)
 
-Detailed installation guides are available in the [wiki](https://www.trinitycore.info/display/tc/Installation+Guide) for
-Windows, Linux and macOS.
+# Install
+Most of the install steps are the same as the TrinityCore ones [here](https://www.trinitycore.info/en/install/Core-Installation).
 
-### MySQL 5.7+
+# Data Files
+To avoid any problems with World of Warcraft we will not be providing the data files for LegionCore instead you can aquire them from [here](https://www.emucoach.com/legion-7-3-5-/6945-repack-7-3-5-legion-wow-repack-wow-legion-7-3-5-repack-blizzlike-fun.html)
```

Comment:
Let's stick to:
`This core has been optimized for DBC/vmap/map/mmaps files from UWoW and are not provided as part of this source code package. Instead the data files for LegionCore can be acquired [here](https://www.emucoach.com/legion-7-3-5-/6945-repack-7-3-5-legion-wow-repack-wow-legion-7-3-5-repack-blizzlike-fun.html).`